src/scss/grommet-core/_objects.layer.scss

```diff
@@ -183,6 +183,16 @@
       }
     }
   }
+
+  .#{$grommet-namespace}tiles {
+    overflow: auto;
+
+    &__more {
+      @include media-query(lap-and-up) {
+        position: relative;
```

Comment:
Can this always be 'relative', rather than just when inside tiles when inside layer?